transaction/std/src/memo/gift_code_sender.rs

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) 2018-2022 The MobileCoin Foundation
+
+//! Object for 0x0002 Gift Code Sender memo type
+//!
+//! This was proposed for standardization in mobilecoinfoundation/mcips/pull/32
+
+use crate::{impl_memo_type_conversions, RegisteredMemoType};
+use mc_transaction_core::MemoError;
+use std::{convert::TryFrom, str};
+
+#[derive(Debug, Clone, Eq, PartialEq, Ord, PartialOrd)]
+pub struct GiftCodeSenderMemo {
```

Comment:
Can you please include a comment explaining what this memo is like you did for the other ones?